### PR TITLE
Fixes recycler not recycling plasma.

### PR DESCRIPTION
**Pull request by harmonyn**
Fixes https://github.com/tgstation/-tg-station/issues/12131.

I SURE DO LOVE THOSE EASY FIXES BOY HOWDY

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
Yay, finally.

File: .gitignore [hunk 1]
```diff
@@ -9,4 +9,4 @@
 *.lk
 *.int
 *.backup
-*.int
\ No newline at end of file
+*.int
```
Comment (Cheridan) [inline .gitignore]: :^)

Comment (bgobandit) [inline .gitignore]: :(

**Comment by phil235** [comment]
>4 commits for a one line change.

**Comment by bgobandit** [comment]
most of them were to get rid of the .int shit

**Pull request merged**